modules/grandpa/src/lib.rs

```diff
@@ -511,6 +514,8 @@ pub(crate) fn find_forced_change<H: HeaderT>(
 }
 
 /// (Re)initialize bridge with given header for using it in external benchmarks.
+// TODO: Will probably need to change this a bit for the finality verifier benchmarks, while keeping
```

Comment:
Do you still need this TODO? Which btw references pallet as finality verifier :)

Reply:
I'm still mulling this over, I'll get back to it eventually :P